testdata/p4_16_samples_outputs/parser-conditional-midend.p4

```diff
@@ -23,7 +19,7 @@ parser p(out bit<32> b) {
     }
     state start_join {
         b = tmp;
-        b = b + 32w1;
+        b = tmp + 32w1;
```

Comment:
We don't remove dead writes to live vars in this pass, only writes to dead vars.  Removing this would require liveness analysis, which is a reverse control flow analysis (local copyprop is forward control flow).

A subsequent liveness and dead code elim pass could eliminate this.